### DISCUSSION TITLE
Monitor Control version compatible with Mac OS High Sierra

### DISCUSSION THREAD

**Discussion by mbirafon** [MonitorControl/MonitorControl#1066] [General]
Hello, 
Is the Monitor Control version compatible with Mac OS High Sierra available?
Can you supply a download link?
Thanks.

**Comment by abstarstrong** [comment]
Hi wanted to second this. Should this be brought up as a issue? Are you planning on having support for older MacOS versions?

**Comment by the0neyouseek** [comment]
Hi,

As is written in the readme https://github.com/MonitorControl/MonitorControl/#macos-compatibility , the latest version compatible with macOs High Sierra (10.13.x) is `v2.1.0` which is quite old but should still work.

You can find the download link in the GitHub releases here : https://github.com/MonitorControl/MonitorControl/releases/tag/v2.1.0

**Comment by mbirafon** [comment]
Hello,
Many thanks.
I have appreciated your help.
Kind regards,
Giambattista Marcucci


Il 14/09/22, 17:22, "Guillaume B." ***@***.***> ha scritto:
